pwa-devdocs/src/technologies/basic-concepts/state-management/index.md

```diff
@@ -56,16 +64,19 @@ and the Redux pattern can be seen in hooks such as [`useQueryResult()`][] and [`
 Currently, PWA Studio abstracts away its Redux implementation details using Peregrine hooks and context providers.
 This opens up the possibility of the project replacing Redux in Peregrine with another state management library without breaking state dependent components, such as those in Venia.
 
-Since Peregrine components require a Redux store, you must create one in your storefront application project and provide reducers:
+PWA Studio allows you to customize reducers and enhancers.
+The following example uses `combineReducers()` to combine the default Peregrine reducers with custom reducers specific to the project and uses the combined reducers when creating the Redux store.
 
 ```jsx
 // Example src/store.js file
 
 import { combineReducers, createStore } from 'redux';
 import { enhancer, reducers } from '@magento/peregrine';
 
+import myReducers from './lib/reducers';
+
 // You can add your own reducers here and combine them with the Peregrine exports.
-const rootReducer = combineReducers(reducers);
+const rootReducer = combineReducers(reducers, myReducers);
```

Comment:
My example was wrong: https://redux.js.org/api/combinereducers

I think the proper example is:

```js
combineReducers({
  ...reducers,
  ...myReducers
})
```